### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by oguh43** [prabeshdhakal/beepy-v1#7]
When the sound finishes playing, the program crashes with Segmentation Fault - Fedora linux

**Comment by jaalmoes** [comment]
Hi, I have the same issue.
To reproduce:
```python
import beepy
beepy.beep(sound="ping")
```
Give the error:
`Erreur de segmentation (core dumped)`

Python 3.12.3 on Archlinux beepy installed with pip

**Comment by ldedier** [comment]
Same here:

```
import beepy
beepy.beep()
```

will throw a

`Segmentation fault: 11  ` on mac

**Comment by PiNuessle** [comment]
Same here on mac, for every permutation of beepy.beep (including sound='name', just the number, empty, and sound=number, for every sound). However, this is only for python 3.12+. My python 3.8.18, 3.8.8, 3.9.20, 3.10.16, and 3.11.11 environments can use it just fine. ~~My suspicion is that there's a called package whose syntax hasn't been updated here.~~

I actually found the issue trying to find an alternative. simpleaudio, the package this one is built on top of, has the segmentation fault. So you can't use simpleaudio at all to fix it. However, you should still be able to print ('\a') to get a very quiet alert sound.

So my advice is, for now, only use this package for environments that use python 3.11 or lower. If you still need a beeping sound because (for instance) you're running a 30 minute compilation proceedure, try the following:

1. In terminal:
`pip3 install PyObjC
pip3 install playsound`
2. grab a public domain .mp3 or .wav file from [commons.wikimedia.org](https://commons.wikimedia.org/w/index.php?search=beep&title=Special:MediaSearch&type=audio&haslicense=unrestricted), such as [this one](https://commons.wikimedia.org/wiki/File:Sputnik_-_Beep.mp3).
3. open your code, and write
`import playsound`
at the top and 
`playsound.playsound('path/to/file')`
after the long function.

And then you have a solution that works for 3.12 and 3.13 on my machine.

tl:dr, bother the simpleaudio devs, because the issue isn't here.

**Comment by prabeshdhakal** [comment]
Hi everyone, thank you for raising the issue and participating in the discourse. 

I am planning on overhauling beepy in a new repository. This means that the current repository will be archived. However, `pip install beepy` will still work :)

I will try to find a more robust dependency for the the new version in order to address the issue mentioned here. If you find time, feel free to make your contribution and create a pull request here: https://github.com/prabeshdhakal/beepy.
